qa/t/webservice_bug_get.t

```diff
@@ -124,6 +123,14 @@ sub post_success {
     );
   }
 
+  # See also to a private bug should not display for the public bug
+  if (!$is_private_bug && !$is_private_user) {
+    ok(
+      !exists $bug->{see_also} || !@{$bug->{see_also}},
+      'See also to a private bug should not display for the public bug and normal user'
+    );
+  }
+
```

Comment:
I'd like to see a few more tests here to cover the whole matrix: public/public, public/private, private/public, and private/private